packages/cdktf-cli/lib/get/generator/emitter/attributes-emitter.ts

```diff
@@ -217,11 +135,15 @@ export class AttributesEmitter {
     if (type.isBoolean) {
       return `this.getBooleanAttribute('${att.terraformName}')`;
     }
+    if (type.isList) {
+      return `(this.getListAttribute('${att.terraformName}') as any) as ${att.type.name}`;
```

Comment:
This will fail since `string[] != attribute type`

Reply:
Removed this part and created #958 to plan a proper solution